core/app/models/workarea/navigation/breadcrumbs.rb

```diff
@@ -13,7 +13,11 @@ def self.from_global_id(via, last: nil)
 
       def initialize(navigable, last: nil)
         @navigable = navigable
-        @last = Taxon.new(name: last) if last.present?
+        @last = if last.try(:navigable?)
```

Comment:
Judging from the build, I think I forgot we'd need a `def navigable?` in the `Navigable` module.